src/object_store/cache.rs

```diff
@@ -351,7 +573,8 @@ impl CachingObjectStore {
                         warn!("Invalidating cache entry for {key:?}; failed writing to a file: {err}");
                         cache.invalidate(&key).await;
```

Comment:
We could also use a counter for these.

Reply:
By these I mean counter for cache file write errors.

Reply:
done